examples/rust/aranya-example-multi-node/src/env.rs

```diff
@@ -0,0 +1,213 @@
+        let admin = devices.pop().expect("expected device");
+        let owner = devices.pop().expect("expected device");
+        Ok(Self {
+            level,
+            passphrase,
+            owner,
+            admin,
+            operator,
+            membera,
+            memberb,
+        })
+    }
+
+    /// Generate environment file.
+    pub async fn generate(&self, path: &Path) -> Result<()> {
+        let mut buf = "".to_string();
+        buf += &format!("export {}={}\r\n", LOG_LEVEL_ENV_VAR, self.level);
+        buf += &format!(
+            "export {}={}\r\n",
+            ONBOARDING_PASSPHRASE_ENV_VAR,
+            self.passphrase.expose_secret()
+        );
+        for device in self.devices() {
+            buf += &format!(
+                "export {}_{}={}\r\n",
+                SYNC_ADDR_ENV_VAR,
+                device.name.to_uppercase(),
+                device.sync_addr
+            );
+            buf += &format!(
+                "export {}_{}={}\r\n",
+                AQC_ADDR_ENV_VAR,
+                device.name.to_uppercase(),
+                device.aqc_addr
+            );
+            buf += &format!(
+                "export {}_{}={}\r\n",
+                TCP_ADDR_ENV_VAR,
+                device.name.to_uppercase(),
+                device.tcp_addr
+            );
+        }
+        fs::write(path, buf).await?;
+        Ok(())
+    }
+
+    /// Set environment variables.
+    pub fn set(&self) {
+        // TODO: set_var() is not safe to call in a multi-threaded program.
+        env::set_var("ARANYA_EXAMPLE", self.level.clone());
+        env::set_var(
+            ONBOARDING_PASSPHRASE_ENV_VAR,
+            self.passphrase.expose_secret(),
+        );
+        for device in self.devices() {
+            env::set_var(
+                format!("{}_{}", SYNC_ADDR_ENV_VAR, device.name.to_uppercase()),
+                device.sync_addr.to_string(),
+            );
+            env::set_var(
+                format!("{}_{}", AQC_ADDR_ENV_VAR, device.name.to_uppercase()),
+                device.aqc_addr.to_string(),
+            );
+            env::set_var(
+                format!("{}_{}", TCP_ADDR_ENV_VAR, device.name.to_uppercase()),
+                device.tcp_addr.to_string(),
+            );
+        }
+    }
+
+    /// Return an Iterator to the list of devices.
+    pub fn devices(&self) -> impl Iterator<Item = &Device> {
+        vec![
+            &self.owner,
+            &self.admin,
+            &self.operator,
+            &self.membera,
+            &self.memberb,
+        ]
+        .into_iter()
+    }
+}
+
+/// Default environment variables.
+impl Default for EnvVars {
+    fn default() -> Self {
+        Self {
+            level: "info".into(),
+            passphrase: "passphrase".into(),
+            owner: Device {
+                name: "owner".into(),
+                aqc_addr: Addr::from_str("127.0.0.1:10000").expect("expected addr"),
```

Comment:
```suggestion
                aqc_addr: Addr::from((Ipv4Addr::LOCALHOST, 10000)),
```

Same for all the others.